pkg/api/http/directmessaging_test.go

```diff
@@ -411,7 +411,7 @@ func TestV1DirectMessagingEndpoints(t *testing.T) {
 		// assert
 		mockDirectMessaging.AssertNumberOfCalls(t, "Invoke", 1)
 		assert.Equal(t, 500, resp.StatusCode)
-		assert.Truef(t, strings.HasPrefix(string(resp.RawBody), "{\"errorCode\":\"ERR_MALFORMED_RESPONSE\",\"message\":\""), "code not found in response: %v", string(resp.RawBody))
+		assert.Truef(t, strings.HasPrefix(string(resp.RawBody), "{\"errorCode\":\"ERR_COMMON_MALFORMED_RESPONSE\",\"message\":\""), "code not found in response: %v", string(resp.RawBody))
```

Comment:
I'm aware of this, and I don't exactly have much of a preference of whether or not to change the errors; It was described to me as breaking but logs are not in the contract of what we have to be super weary of; the old error codes were inconsistent in naming, like:
ERR_RAISE_EVENT_WORKFLOW
ERR_WORKFLOW_NAME_MISSING
...so I envision someday we'd want to rename these errors, unless you see errors like ERR_WORKFLOW_NAME_MISSING not as being prefixed but just a phrase that happens to start with the component.

In the case of the latter, as well as generally just wanting to prevent breaking changes, I can revert to the original error codes.

Reply:
Aren't these the same codes that are returned in http responses too? I haven't looked deeper into the PR yet, this is just something that caught my eye at first glance.

Reply:
Sometimes, yes; in that case, it would be more a potential breaking change if users rely on consistent responses, so it adds another reason to revert to original error codes which I will do so.